recipes/categorical-distance/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "categorical-distance" %}
+{% set version = "1.9" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/categorical-distance-{{ version }}.tar.gz
+  sha256: ae5eaf72048cb4253f6b851f0594a001643574dffc014d03445f9c7c30543a3e
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy
+    - python
```

For noarch need to add Python versions constraints here. Also, since it's checked for in `setup.py`, should probably explicitly list `setuptools`:
```suggestion
    - pip
    - python >=3.6
    - setuptools
  run:
    - numpy
    - python >=3.6
```